spec/mailers/notification_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+describe NotificationMailer do
+  let!(:user) { create(:user, email: "slon@elephant.ru") }
+  let!(:deck) { create(:deck, user: user) }
+  let!(:card) { create(:card, original_text: "слон", translated_text: "elephant", deck: deck) }
+  let(:user_with_counter) { User.for_review_counters_at(Time.now).first }
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.